sdk/tests/conformance2/rendering/attrib-type-match.html

```diff
@@ -115,11 +216,8 @@
     }
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "No GL error from set up");
 
-    debug("");
-    debug("Test vertexAttrib with drawArrays and drawArraysInstanced");
     wtu.setupUnitQuad(gl, 0);
 
-    debug("Correct setup");
```

Comment:
Ah... I just moved the code 118 ~ 119 to line 201 ~ 203, to show that here begins the other part, in addition to my new added part (generic attrib). 

So the "correct setup“ seems to be unnecessary because there is a "No GL error from set up" on line 116/217...